set.mm

```diff
@@ -65983,9 +65981,9 @@ complete lattice has a (least) fixpoint.  Here we specialize this
      universe onto its power class.  Compare ~ canth that works for sets.
 
      This failure comes from a limitation of the collection principle (which is
-     necessary to avoid Russell's paradox ~ ru ): ` P _V ` , being a class,
-     cannot contain proper classes, so ` ~P _V ` , so it is no larger than
-     ` _V ` , which is why ` _I ` "succeeds" in being surjective onto ` ~P _V `
+     necessary to avoid Russell's paradox ~ ru ): ` ~ P _V ` , being a class,
```

Comment:
Why split the `~` from the `P` here?
I suppose it should be `` ` ~P _V ` ``.

Reply:
The main reason to do that is because it's 1:30am !

Reply:
Fixed.